frontend/src/main/app.js

```diff
@@ -68,6 +68,10 @@ store.dispatch(updateIntl({
 const theme = createMuiTheme(Palette)
 
 function App () {
+  useEffect(() => {
```

Comment:
@etfaghaoubeid here we can use `useState` to set the state of loading combined with useEffect and replace the react loading component. So that's what I mean to use a pure react solution, instead of use dom manipulation.